### PR TITLE
Create quick-installerOSX.sh

### PR DESCRIPTION
**Pull request by Pretuer**

### CODE REVIEW

File: quick-installerOSX.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/zsh
```
Comment (LangLangBart) [inline quick-installerOSX.sh]: #!/bin/sh
https://support.apple.com/guide/terminal/about-shell-scripts-apd53500956-7c5b-496b-a362-2845f2aab4bc/mac

File: quick-installerOSX.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+
+echo "Running Installation Script"
+pwd
+cd ~/Library/Application\Support/0ad/mods/
```
Comment (LangLangBart) [inline quick-installerOSX.sh]: cd ~/Library/Application\ Support/0ad/mods/
 missing space

File: quick-installerOSX.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/zsh
+
+
+
+echo "Running Installation Script"
+pwd
```
Comment (LangLangBart) [inline quick-installerOSX.sh]: why do u need "pwd" ?

File: quick-installerOSX.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+cd ~/Library/Application\Support/0ad/mods/
+
+
+git clone https://github.com/LangLangBart/boonGUI.git
+
+echo "installation complete running 0 A.D."
+
+0ad
```
Comment (LangLangBart) [inline quick-installerOSX.sh]: what is this for?

**Review by LangLangBart — CHANGES_REQUESTED** [review]
Could be included under installation of the README file, would look nice with a expand element. 
https://gist.github.com/joyrexus/16041f2426450e73f5df9391f7f7ae5f

File: quick-installerOSX.sh (hunk 3 above)
Comment (Pretuer) [reply quick-installerOSX.sh]: it's redundant i'll delete it

File: quick-installerOSX.sh (hunk 4 above)
Comment (Pretuer) [reply quick-installerOSX.sh]: would've automatically ran the game, but i've removed it

**Comment by LangLangBart** [comment]
Thanks.

**Pull request merged**